libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -71,9 +71,231 @@ const AP_Param::GroupInfo AP_UAVCAN::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ESC_BM", 3, AP_UAVCAN, _esc_bm, 255),
 
+    // @Param: FIX_R
+    // @DisplayName: UAVCAN GNSS Fix broadcast rate
+    // @Description: UAVCAN GNSS Fix broadcast rate in times per second. 0 - disable broadcast.
+    // @Range: 0 20
+    // @User: Advanced
+    AP_GROUPINFO("FIX_R", 4, AP_UAVCAN, _broadcast_fix_rate, 10),
```

Comment:
I'm reluctant to have this broadcast at 10Hz by default. This PR puts 20-30Hz of packets onto the bus when in most cases no one wants that data

Reply:
Agree. It is used only by few MAPIR camera users.